src/com/facebook/buck/android/dalvik/DalvikStatsCache.java

```diff
@@ -32,10 +32,13 @@
   }
 
   DalvikStatsTool.Stats getStats(FileLike entry) {
-    String name = entry.getRelativePath();
-    if (!name.endsWith(".class")) {
+    String[] pathParts = entry.getRelativePath().split("/");
+    String name = pathParts[pathParts.length-1];
+    if (!name.endsWith(".class") || name.equals("module-info.class")) {
```

Comment:
to address https://github.com/facebook/buck/pull/2504#issuecomment-691348177 . this could be changed to 

```
name. endsWith("/module-info.class")
```

Reply:
That's weird, name is the last part of `"META-INF/versions/9/module-info.class"`, which should e exactly `"module-info.class"`.